doc/status-go-changes.md

```diff
@@ -0,0 +1,26 @@
+# How to make changes to status-go
+
+
+## Overview
+
+To make changes to status-go, the best approach is to first identify the issue/feature and the changes to be made directly in status-go.
+For any bugs we want to most likely write a failing test in status-go, and make sure that it passes, before we test on mobile.
+
+Once you are quite confident that the status-go code is ready to be tested on mobile, you can test it locally by running the following command:
+
+```
+env STATUS_GO_SRC_OVERRIDE={your-status-go-directory} make run-{android/ios}
+```
+
+This will recompile status-go and run it on your device.
```

Comment:
I'd rewrite this a bit for clarity: `This will recompile status-go and run status-mobile on your device with that updated status-go version`, wdyt?